fastchat/llm_judge/common.py

```diff
@@ -454,6 +476,32 @@ def chat_compeletion_anthropic(model, conv, temperature, max_tokens):
     return output.strip()
 
 
+def chat_compeletion_palm(chat_state, model, conv, temperature, max_tokens):
+    from fastchat.serve.api_provider import init_palm_chat
+
+    assert model == "palm-2-chat-bison-001"
+
+    if chat_state is None:
+        chat_state = init_palm_chat("chat-bison@001")
+
+    parameters = {
+        "temperature": temperature,
+        "top_p": 0.8,
+        "top_k": 40,
+        "max_output_tokens": max_tokens,
+    }
+    output = API_ERROR_OUTPUT
+    for _ in range(API_MAX_RETRY):
+        try:
+            response = chat_state.send_message(conv.messages[-2][1], **parameters)
+            output = response.text
+            break
+        except Exception as e:
```

Comment:
do not use broad exception

Reply:
I tried to find the doc for `chat_state.send_message`. The possible errors seem not well documented in vertex ai python sdk. Do you have any good idea?
https://cloud.google.com/python/docs/reference/aiplatform/latest/vertexai.preview.language_models.ChatSession#vertexai_preview_language_models_ChatSession_send_message